src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.*;
+
 public class Duke {
+    public static class Task{
```

Comment:
Shouldn't this task class be its own class file? But overall, the coding is neat and easy to understand. Good job .